### DISCUSSION TITLE
How do cycle routes get rendered?

### DISCUSSION THREAD

**Discussion by spaanse** [osmandapp/OsmAnd#17457] [Q&A]
For https://github.com/osmandapp/OsmAnd/issues/17373 I was looking into the render styles.
I am able to create a style that defocuses the car roads.
However, I now want to see the cycle routes at lower zoom levels.
I see that their styles are defined in `routes.addon.render.xml`
However, that file seems inconsistent with how the routes get rendered
For me: `rcn` routes are colored a pink/purple and `ncn` routes are orange.
However `routes.addon.render.xml` specifies that they should be bright green and translucent bright green respectively.

I have the following questions:
- What does decide how these get rendered?
- Is it possible to select between `routes.addon.render.xml` and `routes-bicycle-focus.addon.render.xml` like with regular map styles? Or should The rendering of routes be incorporated into a non-addon render style instead?
- Why does changing the minzoom in `routes.addon.render.xml` not change whether the routes are rendered at lower zooms?

**Comment by mikehgentry** [comment]
Maybe this does what you want, or gives you some ideas how to do it?

https://github.com/Louis-LJ/OsmAnd_Cycling_Map

**Reply by spaanse** [reply]
Thanks for the link.
It indeed changes the colors of the cycle routes and makes them more visible.
It still has the issue that the routes are not visible when zoomed out (zoom <= 10)
